src/Exception/InvalidIfExpressionException.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sofascore\PurgatoryBundle\Exception;
+
+use Symfony\Component\ExpressionLanguage\Expression;
+use Symfony\Component\ExpressionLanguage\SyntaxError;
+
+final class InvalidIfExpressionException extends RuntimeException
```

Comment:
@Brajk19 I'm thinking, wouldn't `InvalidArgumentException` be more appropriate?

Reply:
agreed https://github.com/sofascore/purgatory-bundle/pull/101/commits/1f4a8a34c9f5dc0dc715500b4c9d7759c43d4917